src/Engine/Renderer/RenderTechnique/ShaderProgram.cpp

```diff
@@ -317,65 +321,63 @@ void ShaderProgram::setUniform( const char* name, std::vector<float> values ) co
 //!
 
 void ShaderProgram::setUniform( const char* name, const Core::Vector2i& value ) const {
-    m_program->setUniform( name, Core::toGlm( value ) );
+    m_program->setUniform( name, value );
```

Comment:
so is file `Core/Math/GlmAdapters` still needed?

Reply:
Yes as already discussed some time ago.